Tests/HtmlPrettyPrintTests/PrettyTests.swift

```diff
@@ -34,8 +38,15 @@ class PrettyTests: XCTestCase {
        Let's see what happens! -->
   <div>
     <div id="some-long-id"
-         class="foo bar baz"
-         style="color: red;">
+         class="foo
+                bar
+                baz
+                class1
+                class2
+                class3"
+         style="color: red;
+                background: blue;
+                padding: rem(2);">
```

Comment:
p cool! had these styles/classes fit on one line there would be no newlines. but once they break, they all align together.